### PR TITLE
CON-1085: Removed the hello-world tutorial

### PR DESCRIPTION
**Pull request by shamsasari**
It has been moved to https://github.com/R3Conclave/conclave-tutorials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,6 @@ This sample allows bidders to confidentially submit their bids to an enclave (A
 ### [Conclave Corda Trade](./conclave-corda-trade):
 This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes serve as clients which send encrypted orders from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
 
-### [Conclave Hello World](./hello-world):
-This is a simple conclave application which shows you how you can reverse a string in java.
-
 ### [Conclave Hello World Python](./hello-world-python):
```
Comment (ac101m) [inline README.md:40]: Should we not remove this as well?

**Review by ac101m — COMMENTED** [review]
Do we not need to get rid of the python hello world too?

File: README.md (hunk 1 above)
Comment (shamsasari) [reply README.md:40]: That was added by @snedamle recently. It's not referenced by the Docsite and separate to the hello world.

Comment (anien) [reply README.md:40]: I believe Sneha was adding hello world sample with python to show how python can be used in Conclave. It's DevRel's sample.

**Comment by anien** [comment]
LGTM!

**Comment by shamsasari** [comment]
Any objections @bonifazi-r3 and @filipesoliveira? Otherwise, if someone could approve.

**Comment by bon000** [comment]
Not really sure what is to review here, lol. Will this go to `release-1.3` too?

**Comment by filipesoliveira** [comment]
LGTM

**Review by bon000 — APPROVED** [review]

**Comment by shamsasari** [comment]
>Will this go to release-1.3 too?

Correct.

**Pull request merged**